railties/lib/rails/generators/rails/app/templates/config/secrets.yml

```diff
@@ -12,8 +12,8 @@
 
 # Shared secrets are available across all environments.
 
-shared:
-  api_key: 123
+# shared:
+#   api_key: <%= app_secret %>
```

Comment:
I'd use a human readable example here since that ` Make sure the secret is at least 30 characters and all random` comment is about the app secret not about a shared api key.